.travis.yml

```diff
@@ -50,16 +49,18 @@ install:
 script:
   - export PIPENV_IGNORE_VIRTUALENVS=1
   # for some reason the migrations check is failing only on django 2.0
-  - if [[ $TRAVIS_PYTHON_VERSION == 3.5 ]]; then pipenv run ./tests/manage.py makemigrations config pki --dry-run | grep "No changes detected"; fi
-  - pipenv run coverage run --source=openwisp_controller runtests.py
-  - |
-    if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
-      # gets commit message of last commit before pull request merge
-      COMMIT_MESSAGE=$(git log $TRAVIS_PULL_REQUEST_SHA --format=%B -n 1)
-      printf "Checking commit message:\n\n"
-      printf "$COMMIT_MESSAGE\n\n"
-      pipenv run checkcommit --message "$COMMIT_MESSAGE"
+  - >
+    if [[ $TRAVIS_PYTHON_VERSION == 3.5 ]]; then
+    pipenv run openwisp-utils-qa-checks 
```

Comment:
Also, can you please fix the indentation? 
```
    if [[ $TRAVIS_PYTHON_VERSION == 3.5 ]]; then
    pipenv run openwisp-utils-qa-checks 
```
should be:
```
```
    if [[ $TRAVIS_PYTHON_VERSION == 3.5 ]]; then
        pipenv run openwisp-utils-qa-checks 
```
Just for readability! :smile:
```